tremor-connectors/src/utils/tls.rs

```diff
@@ -167,22 +157,20 @@ impl TLSClientConfig {
             let mut reader = BufReader::new(certfile);
 
             let cert = rustls_pemfile::read_one(&mut reader)?.and_then(|item| match item {
-                Item::X509Certificate(cert) => Some(Certificate(cert)),
+                Item::X509Certificate(cert) => Some(cert),
                 _ => None,
             });
-            cert.and_then(|cert| roots.add(&cert).ok())
+            cert.and_then(|cert| roots.add(cert).ok())
                 .ok_or_else(|| Error::InvalidCertificate(cafile.to_string_lossy().to_string()))?;
             roots
         } else {
             SYSTEM_ROOT_CERTS.clone()
         };
 
-        let tls_config = ClientConfig::builder()
-            .with_safe_defaults()
-            .with_root_certificates(roots);
+        let tls_config = ClientConfig::builder().with_root_certificates(roots);
```

Comment:
same as above, safe_default soiunds really good vs no safe defaults :D

Reply:
you removed the safe defaults you did :D I agree it makes no sense, lets not remove it

Reply:
So, this was renamed in `rustls` but not sure what else has changed:

```
error[E0599]: no method named `with_safe_default_protocol_versions` found for struct `ConfigBuilder<ServerConfig, WantsVerifier>` in the current scope
  --> tremor-connectors/src/utils/tls.rs:72:14
   |
71 |           let server_config = ServerConfig::builder()
   |  _____________________________-
72 | |             .with_safe_default_protocol_versions()
   | |             -^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ method not found in `ConfigBuilder<ServerConfig, WantsVerifier>`
   | |_____________|
   |
   |
   = note: the method was found for
           - `ConfigBuilder<S, WantsVersions>`
```

These may no longer be needed at all judging by: https://github.com/rustls/rustls/issues/1372

Reply:
Confirmed. https://docs.rs/rustls/latest/rustls/server/struct.ServerConfig.html#method.builder We no longer need the safe defaults are they are defaulted. Same for client...